cmd/helm/history.go

```diff
@@ -90,25 +104,60 @@ func (cmd *historyCmd) run() error {
 		return nil
 	}
 
-	fmt.Fprintln(cmd.out, formatHistory(r.Releases, cmd.colWidth))
+	releaseHistory := getReleaseHistory(r.Releases)
+
+	var history []byte
+	var formattingError error
+
+	switch cmd.outputFormat {
+	case "yaml":
+		history, formattingError = yaml.Marshal(releaseHistory)
+	case "json":
+		history, formattingError = json.Marshal(releaseHistory)
+	default:
```

Comment:
Unknown values should probably cause an error. Otherwise adding a new format is a breaking change.